### PR TITLE
feat: added Get Computed Label endpoint

### LINKED ISSUE #195: Implement Get Computed Label endpoint
**What would you like to be added**:
[Get Computed Label endpoint](https://www.w3.org/TR/webdriver/#dfn-get-computed-label)

**Why is this needed**:
It's a remainding endpoint.

**Related:**
https://www.npmjs.com/package/dom-accessibility-api
https://www.npmjs.com/package/aria-api

### PR DESCRIPTION
**Pull request by MeisterRed**
- Added Get Computed Label Endpoint

- Added Get Computed Label Test

This pull request focuses on the Get Computed Label Endpoint. I've tested everything on my end and it seems to be working fine. Please let me know if anything needs to be changed or if the test file needs to be improved

Closes #195

### CODE REVIEW

File: src/WebElement/WebElement.ts [hunk 1]
```diff
@@ -397,6 +398,10 @@ class WebElement {
 
     return selected;
   }
+
+  getLabel(): string {
```
Comment (poftadeh) [inline src/WebElement/WebElement.ts:402]: Please add a JSDoc for this.

File: test/jest/e2e/get-computed-label.test.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+const request = require('supertest');
+const nock = require('nock');
+
+const { app } = require('../../../build/app');
+const { createSession } = require('./helpers');
+const { COMMANDS, ELEMENT } = require('../../../build/constants/constants');
+const { iterInternalSymbol } = require('jsdom/lib/jsdom/living/generated/utils');
```
Comment (poftadeh) [inline test/jest/e2e/get-computed-label.test.js:7]: Unused?

File: test/jest/e2e/get-computed-label.test.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        .send({ using: 'css selector', value: 'button' })
+
+      const {
+        body: { value },
+      } = await request(app)
+          .get(`/session/${sessionId}/element/${elementId}/computedlabel`)
+
+      expect(value).toBe('Close');
+    });
+});
```
Comment (poftadeh) [inline test/jest/e2e/get-computed-label.test.js:54]: nit: newline

**Review by poftadeh — COMMENTED** [review]
Good so far, just need to fix the conflicts.

**Review by poftadeh — COMMENTED** [review]
Why is the diff showing 93 file changes? Rebase your commits on top of `origin/master`.

**Review by poftadeh — APPROVED** [review]

**Review by pynnl — APPROVED** [review]

**Review by manekenpix — APPROVED** [review]

**Review by zthaver — APPROVED** [review]

**Pull request merged**